auth-api/src/main/java/io/stargate/auth/api/impl/AuthApiRunner.java

```diff
@@ -36,6 +36,6 @@ public AuthApiRunner(
   public void start() {
     AuthApiServer authApiServer =
         new AuthApiServer(this.authenticationService, this.metrics, this.httpMetricsTagProvider);
-    authApiServer.run("server", "config.yaml");
+    authApiServer.run("server", "authapi-config.yaml");
```

Comment:
Mentioned in description: change `config.yaml` to service-specific to simplify overrides (can use same name and there is no confusion on what is used where) and as minor documentation. There is no benefit from having same name in the first place.